style.css

```diff
@@ -12957,6 +12877,80 @@ ul.download-versions {
   margin-bottom: 0;
 }
 
+.nhsuk-bordered-tabs-container {
+  padding-top: 8px;
+  background-color: #ffffff;
+  border-bottom: 2px solid #aeb7bd;
```

Comment:
Hardcoded colours? shall we use NHSUK framework ones for accessibility standards

Reply:
This is the output css. It is generated from nhsuk vars.